.github/ISSUE_TEMPLATE/feature_request.md

```diff
@@ -0,0 +1,20 @@
+---
+name: Feature request
+about: Suggest an idea for this project
+title: "[FEATURE_REQUEST]"
+labels: feature request
+assignees: ''
+
+---
+
+**Is your feature request related to a problem? Please describe.**
+A clear and concise description of what the problem is. Ex. I'm always frustrated when [...]
```

Comment:
Related to the comment above:

A clear and concise description of what the problem is. Ex. Almost every time I make use of `useOthers` I need to check if [...]

Reply:
I've updated the template with the `Room.id` scenario to illustrate what kind of information we are waiting for.